bin/questions.js

```diff
@@ -21,6 +19,92 @@ const QUESTIONS = [
             'componentDidCatch'
         ]
     }
```

Comment:
And this?

Reply:
What's wrong here?

Reply:
Ignore me here... can't find any issues now, so guess I was just tired last night, lol.  Sorry about that!